gcloud-java-examples/src/main/java/com/google/gcloud/examples/dns/snippets/ManipulateZonesAndRecordSets.java

```diff
@@ -23,6 +23,7 @@
 package com.google.gcloud.examples.dns.snippets;
 
 import com.google.gcloud.dns.ChangeRequest;
+import com.google.gcloud.dns.ChangeRequestInfo;
```

Comment:
Looks like technically it is not the case of "unnecessary import" as the `ChangeRequest` is being used.
However, I don't think it is used correctly `ChangeRequest.Status` as it should be `ChangeRequestInfo.Status` (and if you want statically imported).

Also, I wonder if there shouldn't be a `reload` method to `ChangeRequest`?
If so, snippets and README could be changed a little to actually use `ChangeRequest`.

The associated issue, #679, is actually suggesting an `isDone`  which can also make the usage and examples nicer.